Makefile

```diff
@@ -40,6 +40,12 @@ all: copr-rpms rpms
 local:
 	$(MAKE) RPM_DIST="0.$(shell date '+%s')" all
 
+check:
+	black chroma_core/ chroma_api/ chroma_cli/ tests/
+	cargo fmt
+	PQ_LIB_DIR=/usr/pgsql-9.6/lib cargo check
+	PQ_LIB_DIR=/usr/pgsql-9.6/lib cargo clippy -- -W warnings
```

Comment:
This will only work for the first compile. There is a bug where clippy doesn't properly report errors when doing incremental compilation.

Reply:
It looks fixed now.